packages/kbn-expandable-flyout/README.md

```diff
@@ -9,30 +9,34 @@ The flyout is composed of 3 sections:
 - a left wider section to show more details
 - a preview section, that overlays the right section. This preview section can display multiple panels one after the other and displays a `Back` button
 
-At the moment, displaying more than one flyout within the same plugin might be complicated, unless there are in difference areas in the codebase and the contexts don't conflict with each other.
+## Design decisions
```

Comment:
Hey @PhilippeOberti ,

1. I remember you were testing this component in Storybook so it will be great to mention storybook and if somebody wants to implement the flyout, how can they play with it in Storybook.
2. Secondly, it will be great to have some wireframe/screenshot here so that people can imagine what is left/right/preview panel.

Reply:
great idea. I wasn't sure it'd be easy to setup Storybook for a package but it's exactly the same as a plugin. Do you think having Storybook would be enough, or addin g a screenshot would be useful here?


https://github.com/elastic/kibana/assets/17276605/3b5c0aca-6e6e-4f74-9ac8-358edb478083

Reply:
This is great. I think just story book instructions should be fine as well. This will make adoption much easier.